packages/integration-sdk-cli/src/commands/types.ts

```diff
@@ -0,0 +1,154 @@
+import * as log from '../log';
+import { loadConfig } from '../config';
+import * as path from 'path';
+import { buildStepDependencyGraph } from '@jupiterone/integration-sdk-runtime';
+import { createCommand } from 'commander';
+import {
+  IntegrationStepExecutionContext,
+  Step,
+  StepGraphObjectMetadataProperties,
+  StepEntityMetadata,
+  StepRelationshipMetadata,
+} from '@jupiterone/integration-sdk-core';
+import { promises as fs } from 'fs';
+
+export interface TypesCommandArgs {
+  projectPath: string;
+  typesFilePath: string;
+}
+
+export function types() {
+  return createCommand('types')
```

Comment:
I kinda wish this and `visualize` would output their content to the console unless provided the `-o, --output-path` option.